integration/SimpleCacheClient.test.ts

```diff
@@ -67,103 +74,100 @@ describe('SimpleCacheClient.ts Integration Tests', () => {
     );
     await momento.deleteCache(cacheName);
   });
-  // TODO: deprecating credentials file for now
-  // it('should use the default auth token from ~/.momento/credentials.toml', async () => {
-  //   createSystemCredentials();
-  //   const cacheName = v4();
-  //   const momento = new Momento();
-  //   await momento.createCache(cacheName);
-  //   await expect(momento.createCache(cacheName)).rejects.toThrow(
-  //     AlreadyExistsError
-  //   );
-  //   await momento.deleteCache(cacheName);
-  //
-  //   removeSystemCredentials();
-  // });
-  //
-  // it('should use the MOMENTO_PROFILE auth token from ~/.momento/credentials.toml', async () => {
-  //   createSystemCredentials('profile2');
-  //   const cacheName = v4();
-  //   const momento = new Momento();
-  //   await momento.createCache(cacheName);
-  //   await expect(momento.createCache(cacheName)).rejects.toThrow(
-  //     AlreadyExistsError
-  //   );
-  //   await momento.deleteCache(cacheName);
-  //   removeSystemCredentials();
-  // });
+
+  it('should use the default auth token from ~/.momento-test/credentials', async () => {
+    createSystemCredentials();
+    const cacheName = v4();
+    const momento = new SimpleCacheClient(AUTH_TOKEN, 1111);
+    await momento.createCache(cacheName);
+    await expect(momento.createCache(cacheName)).rejects.toThrow(
+      AlreadyExistsError
+    );
+    await momento.deleteCache(cacheName);
+
+    removeSystemCredentials();
```

Comment:
We probably want to put this in like a try/finally so that we always remove the credentials even if there was an error in the test?